### PR TITLE
Pylint api.pubsub

### PR DESCRIPTION
**Pull request by gctucker**
Fix Pylint warnings in api.pubsub and run it in GitHub Actions

### CODE REVIEW

File: api/pubsub.py [hunk 1]
```diff
@@ -3,40 +3,54 @@
...
 
     async def _init_sub_id(self):
         await self._redis.setnx(self.ID_KEY, 0)
 
     async def subscribe(self, channel):
+        """Subscribe to a Pub/Sub channel
+
+        Subscribe to a given channel and return a Subscription object
+        containing the subscription id which can then be used again in other
+        methods.
```
Comment (JenySadadia) [inline api/pubsub.py:52]: I would suggest,
```     
        Subscribe to a given channel asynchronously and return a Subscription object
        containing the subscription id and channel which can then be used again in other
        methods.
```

File: api/pubsub.py [hunk 2]
```diff
@@ -45,6 +59,11 @@ async def subscribe(self, channel):
             return Subscription(id=sub_id, channel=channel)
 
     async def unsubscribe(self, sub_id):
+        """Unsubscribe from a Pub/Sub channel
+
+        Unsubscribe from a channel using the provided subscription id as found
+        in a Subscription object.
```
Comment (JenySadadia) [inline api/pubsub.py:65]: I would suggest,
```
        Unsubscribe from a channel asynchronously using the provided subscription id as found
        in a Subscription object and return True. Return False if subscription id can not be found 
        in `_subscriptions` dictionary. 
```

**Review by JenySadadia — COMMENTED** [review]
I have submitted a few suggestions for the docstring of subscribe and unsubscribe methods.
LGTM otherwise.

File: api/pubsub.py (hunk 1 above)
Comment (gctucker) [reply api/pubsub.py:52]: Only the id is needed for other methods in this class, which is why I didn't mention the channel explicitly.  It's part of the Subscription model, which is mentioned as the type of the returned object.

File: api/pubsub.py (hunk 2 above)
Comment (gctucker) [reply api/pubsub.py:65]: The `_subscriptions` is a private attribute of the class, so it's an implementation detail and not something that should be shared with users.  But yes, returning True / False should be documented (or maybe the method should raise an exception).  So I'll fix that.

Comment (JenySadadia) [reply api/pubsub.py:77]: @gctucker I think this is not resolved yet.

**Review by JenySadadia — COMMENTED** [review]
@gctucker I think the docstring for `unsubscribe` is not resolved yet.

**Comment by gctucker** [comment]
It should be resolved in #105.  It became a larger change with exceptions so I moved it to a separate PR as this wasn't related to fixing Pylint errors any more.

**Comment by gctucker** [comment]
@JenySadadia Could you please approve this PR unless you have any other comments?  That way we could have it merged and rebase #105 as at the moment it's based on top of this one.

**Comment by JenySadadia** [comment]
> @JenySadadia Could you please approve this PR unless you have any other comments? That way we could have it merged and rebase #105 as at the moment it's based on top of this one.

Sure.

**Review by JenySadadia — APPROVED** [review]

**Pull request merged**